script.js

```diff
@@ -0,0 +1,59 @@
+const savedList = document.getElementById('savedList');
+const doneList = document.getElementById('doneList');
+let savedListTitle = document.getElementById('savedListTitle');
+savedListTitle.innerHTML = '해야할 일';
+doneListTitle.innerHTML = '완료한 일';
+
+form.addEventListener('submit', (e) => {
+  //할 일 개수 세기
+  const countSavedNum = () => {
+    let savedNum = savedList.getElementsByTagName('li').length;
+    savedListTitle.innerHTML = `해야할 일(${savedNum})`;
+  };
+
+  const countDoneNum = () => {
+    let doneNum = doneList.getElementsByTagName('li').length;
+    doneListTitle.innerHTML = `완료한 일(${doneNum})`;
+  };
+
+  //할 일 구현
+  const li = document.createElement('li');
+  li.innerText = document.getElementById('input').value;
+  li.id = li.innerText;
+  li.isDone = false;
```

Comment:
```suggestion
  li.innerText = document.getElementById('input').value;
  li.id = li.innerText;
  document.getElementById('input').value = '';
  li.isDone = false;
```
이런 식으로 할 일을 입력한 뒤 input 값을 비워 주는 코드도 추가하면 좋을 것 같아요~

Reply:
디테일한 부분 의견 감사드립니다!!